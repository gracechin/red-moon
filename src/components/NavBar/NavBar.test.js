import { render, screen } from "@testing-library/react";
import { NavBar } from "./index";

describe("<NavBar />", () => {
  it("should render component", () => {
    render(<NavBar />);
    expect(screen.getByTestId("navbar")).toMatchSnapshot();
  });

  it("should have website name", () => {
    render(<NavBar />);
    expect(screen.getByTestId("navbar")).toHaveTextContent("Red Moon");
  });
});
