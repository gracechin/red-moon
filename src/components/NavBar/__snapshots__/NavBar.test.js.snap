// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> should render component 1`] = `
<div
  data-testid="navbar"
>
  <h1>
    Red Moon
  </h1>
</div>
`;
